p2p/types/p2p_config.go

```diff
@@ -0,0 +1,12 @@
+package types
+
+import "github.com/pokt-network/pocket/shared/modules"
+
+var _ modules.P2PConfig = &P2PConfig{}
+
+func (x *P2PConfig) IsEmptyConnType() bool {
```

Comment:
Why do we need this interface if it can just be part of the protobuf?

Reply:
Protobuf doesn't automatically generate this custom logic. This is used to support the migration from shared. There's a TODO to convert this back to an enum